src/api.js

```diff
@@ -596,32 +610,39 @@ class VLWholeSlideMicroscopyImageViewer {
    */
   activateDrawInteraction(options) {
     this.deactivateDrawInteraction();
-    const freehand = options.freehand ? options.freehand : false;
+    
     const customOptionsMapping = {
       point: {
         type: 'Point',
+        geometryName: 'Point'
       },
       circle: {
         type: 'Circle',
+        geometryName: 'Circle'
       },
       box: {
         type: 'Circle',
+        geometryName: 'Box',
         geometryFunction: createRegularPolygon(4),
       },
-      polygon: {
+      polyline: {
```

Comment:
I don't think we need to change the property name

Reply:
done